pages/transactional-email/how-to/manage-tem-plans.mdx

```diff
@@ -0,0 +1,82 @@
+---
+meta:
+  title: How to manage TEM plans
+  description: Guide to managing your TEM plans with Scaleway, including upgrading, downgrading, and monitoring usage.
+content:
+  h1: How to manage TEM plans
+  paragraph: Step-by-step guide to managing your TEM plans with Scaleway.
+tags: tem plans management transactional email
+dates:
+  validation: 2024-03-12
+  posted: 2024-03-12
+categories:
+  - transactional-email
+---
+
+Plans are Transactional Email tiers of service. Each plan includes a set of features, pricing, and usage limits tailored to meet your use cases, ranging from low-volume occasional senders to high-volume enterprise users. Plans are scoped to Projects.
+
+Two plans are available:
+  - **Essential** - pay-as-you-go plan designed for occasional or low-volume email sending needs. This plan offers simplified management and a single webhook per domain.
+  - **Scale** - fixed-price plan tailored for high-volume email sending. Includes 100K emails. Additional emails are charged. This plan includes a dedicated IP address and unlimited webhooks per domain. It also features advanced blocklist management, allowing for manual additions.
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/iam/concepts/#owner) status or [IAM permissions](/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- [Configured your domain(s) with Transactional Email](/transactional-email/how-to/configure-domain-with-transactional-email/)
+- Added [SPF, DKIM](/transactional-email/how-to/add-spf-dkim-records-to-your-domain/), [MX](/transactional-email/how-to/add-mx-record-to-your-domain/) and [DMARC](/transactional-email/how-to/add-dmarc-record/) records to your domain(s)
```

Comment:
```suggestion
```
A domain is not needed to configure a Plan (in fact, a plan is chosen before any domain creation)